src/preview.cpp

```diff
@@ -939,9 +938,8 @@ namespace {
 
         // create decoding table
         unsigned long invalid = 64;
-        unsigned long decodeBase64Table[256] = {};
-        for (unsigned long &i : decodeBase64Table)
-            i = invalid;
+        unsigned long decodeBase64Table[256];
```

Comment:
This is a good candidate to be transformed to `std::array`. Then we could also use `decodeBase64Table.fill(...)`

Reply:
I tried that. Didn't want to compile under Windows.